Sources/Brave/Frontend/Browser/Search/SearchEngines.swift

```diff
@@ -192,8 +190,8 @@ public class SearchEngines {
     disabledEngineNames.removeValue(forKey: engine.shortName)
   }
 
-  func disableEngine(_ engine: OpenSearchEngine) {
-    if isEngineDefault(engine) {
+  func disableEngine(_ engine: OpenSearchEngine, isPrivateBrowsing: Bool) {
+    if isEngineDefault(engine, type: isPrivateBrowsing ? .privateMode : .standard) {
```

Comment:
Why not just make the engine type the argument directly instead of `isPrivateBrowsing` boolean like you did for the other methods here?

Reply:
Fixed